php/WordPressPlugin.php

```diff
@@ -36,7 +36,7 @@ public function __construct(string $pluginFile)
 		/*
 		 * setupControllers after plugins_loaded so wp-includes/pluggable.php is loaded
 		 */
-		add_action('plugins_loaded', [$this, 'setupControllers']);
+		add_action('init', [$this, 'setupControllers']);
```

Comment:
Wasn't the `plugins_loaded` hook point necessary for something else?

Reply:
Robbin added this, so I asked him to answer this comment tomorrow.

Reply:
plugins_loaded was necessary after we added the current_user_can('administrator') check (because at that point wp-includes/pluggable.php is loaded which contains the function. See PR #125 ). In this PR the i18n functions are called with load_plugin_textdomain but these aren't available at this point (plugins_loaded). The "init" hook seemed the best first candidate in the hooks orders. In the WordPress codex this is noted at the "init" hook: "Typically used by plugins to initialize. The current user is already authenticated by this time". See https://codex.wordpress.org/Plugin_API/Action_Reference for the full list.